README.md

```diff
@@ -106,6 +106,8 @@ TAG Contributor Strategy has working groups and teams to effectively collaborate
 
 Status: [Proposed](https://github.com/cncf/tag-contributor-strategy/issues/365)
 
+### Contributor Strategy Overview Diagram
+![TAG-CS Roadmap visuals](https://github.com/Riaankl/tag-contributor-strategy/assets/61125752/3282800b-5c61-431e-bcde-de2d6bd3afc2)
```

Comment:
Rather than having this image in your GH account, we should probably move it to somewhere in the TAG Contributor Strategy repo. Maybe in https://github.com/cncf/tag-contributor-strategy/tree/main/website/static/images, which is where we have the logo? 

@aliok Is this the best place to put the WG diagram or is there somewhere better?

Reply:
You are correct @geekygirldawn . I moved it to the same location as the logo.

Reply:
Current location is good, thanks!